### PR TITLE
LPS-162046 Category Facet ordering

### PR DESCRIPTION
**Pull request by felipelorenz**
https://issues.liferay.com/browse/LPS-162046

### CODE REVIEW

**Comment by felipelorenz** [comment]
ci:test:relevant

**Comment by felipelorenz** [comment]
ci:test:sf

**Comment by felipelorenz** [comment]
ci:test:search

**Comment by felipelorenz** [comment]
ci:test:relevant

**Comment by felipelorenz** [comment]
ci:test:relevant

**Comment by felipelorenz** [comment]
ci:test:relevant

File: modules/apps/portal-search/portal-search-web/src/test/java/com/liferay/portal/search/web/internal/facet/display/context/BaseCategoriesSearchFacetDisplayContextTestCase.java [hunk 1]
```diff
@@ -246,6 +248,178 @@ public void testOneTermWithPreviousSelection() throws Exception {
...
+				StringPool.BLANK, "count:asc");
+
+		List<BucketDisplayContext> bucketDisplayContexts =
+			assetCategoriesSearchFacetDisplayContext.getBucketDisplayContexts();
+
+		String nameFrequencyString = _buildNameFrequencyString(
+			bucketDisplayContexts);
+
+		Assert.assertEquals(
+			bucketDisplayContexts.toString(), "3:3|4:4|2:5|1:6",
```
Comment (gustavolimav) [inline modules/apps/portal-search/portal-search-web/src/test/java/com/liferay/portal/search/web/internal/facet/display/context/BaseCategoriesSearchFacetDisplayContextTestCase.java:274]: Make assert more clear.

File: modules/apps/portal-search/portal-search-web/src/test/java/com/liferay/portal/search/web/internal/facet/display/context/BaseCategoriesSearchFacetDisplayContextTestCase.java [hunk 2]
```diff
@@ -246,6 +248,178 @@ public void testOneTermWithPreviousSelection() throws Exception {
...
+				StringPool.BLANK, "count:desc");
+
+		List<BucketDisplayContext> bucketDisplayContexts =
+			assetCategoriesSearchFacetDisplayContext.getBucketDisplayContexts();
+
+		String nameFrequencyString = _buildNameFrequencyString(
+			bucketDisplayContexts);
+
+		Assert.assertEquals(
+			bucketDisplayContexts.toString(), "1:6|2:5|4:4|3:3",
```
Comment (gustavolimav) [inline modules/apps/portal-search/portal-search-web/src/test/java/com/liferay/portal/search/web/internal/facet/display/context/BaseCategoriesSearchFacetDisplayContextTestCase.java:318]: same

File: modules/apps/portal-search/portal-search-web/src/test/java/com/liferay/portal/search/web/internal/facet/display/context/BaseCategoriesSearchFacetDisplayContextTestCase.java [hunk 3]
```diff
@@ -246,6 +248,178 @@ public void testOneTermWithPreviousSelection() throws Exception {
...
+				StringPool.BLANK, "key:asc");
+
+		List<BucketDisplayContext> bucketDisplayContexts =
+			assetCategoriesSearchFacetDisplayContext.getBucketDisplayContexts();
+
+		String nameFrequencyString = _buildNameFrequencyString(
+			bucketDisplayContexts);
+
+		Assert.assertEquals(
+			bucketDisplayContexts.toString(), "1:3|2:1|3:4|4:2",
```
Comment (gustavolimav) [inline modules/apps/portal-search/portal-search-web/src/test/java/com/liferay/portal/search/web/internal/facet/display/context/BaseCategoriesSearchFacetDisplayContextTestCase.java:361]: same

**Comment by felipelorenz** [comment]
ci:test:sf

**Comment by felipelorenz** [comment]
ci:test:search

**Comment by felipelorenz** [comment]
ci:test:relevant

**Comment by felipelorenz** [comment]
ci:test:relevant

**Comment by joshuacords** [comment]
Continuing here: https://github.com/liferay-search/liferay-portal/pull/705

**Pull request closed without merging**